src/App.tsx

```diff
@@ -1,5 +1,56 @@
+const colors = [
+    {
+        hex: 'EFDECD',
+        name: 'Almond',
+        comp: [
+            { hex: 'ADC6E0', name: 'Periwinkle Gray' },
+            { hex: 'D8E6F5', name: 'Link Water' },
+            { hex: 'FFFFFF', name: 'White' },
+        ],
+    },
+    {
+        hex: 'CD9575',
+        name: 'Antique Brass',
+        comp: [
+            { hex: '649DBE', name: 'Cerulean Frost' },
+            { hex: '88BCDA', name: 'Half Baked' },
+            { hex: 'EFC9B3', name: 'Desert Sand' },
+        ],
+    },
+];
+
 const App = () => {
-    return <h1>Hello World</h1>;
+    return (
+        <div style={{ padding: '20px', fontFamily: 'Arial, sans-serif' }}>
+            <h1>Colors</h1>
+            <div style={{ display: 'flex', flexWrap: 'wrap' }}>
+                {colors.map(color => (
+                    <div
+                        key={color.hex}
+                        style={{
+                            margin: '10px',
+                            padding: '10px',
+                            backgroundColor: `#${color.hex}`,
```

Comment:
The text does not contrast well with the background color in all cases. You're displaying a variety of colors, so you could:
- use different text colors that contrast the primary color
- change the design so that the text is always on the same background, and you display colors beside them